distributed/tests/test_core.py

```diff
@@ -559,6 +559,31 @@ def ping(comm, delay=0.1):
     rpc.close()
 
 
+@gen_test()
+def test_connection_pool_respects_limit():
```

Comment:
Sure, I'll change it (the first version even used `async def`, but I changed it back for consistency with the surrounding code ;-) )

Reply:
I now opted for converting the whole test module (`test_core.py`) to the `async def` style, so at least this one file is consistent concerning style ...

Reply:
> I now opted for converting the whole test module (test_core.py) to the async def style, so at least this one file is consistent concerning style ...

Great!  Thanks!  Now I hope that you submit PRs that include single tests in all of the other files, so that you fix them too :)